diff --git a/CMakeLists.txt b/CMakeLists.txt
index d04e2f2..7273537 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,13 +19,13 @@ find_package(PCL 1.7 REQUIRED)
 find_package(OpenCV REQUIRED)
 find_package(Boost REQUIRED)
 find_package(ITK REQUIRED)
-find_package(VTK REQUIRED)
+#find_package(VTK REQUIRED)
 find_package(OpenGL REQUIRED)
-find_package(ACVD REQUIRED)
+#find_package(ACVD REQUIRED)
 
 include(${QT_USE_FILE})
 include(${ITK_USE_FILE})
-include(${VTK_USE_FILE})
+#include(${VTK_USE_FILE})
 add_definitions(${QT_DEFINITIONS})
 
 include_directories(${Boost_INCLUDE_DIR} ${PCL_INCLUDE_DIRS} ${OpenCV_INCLUDE_DIRS} ${ACVD_INCLUDE_DIRS})
diff --git a/texturing/CMakeLists.txt b/texturing/CMakeLists.txt
index a8b22ff..c7359ca 100644
--- a/texturing/CMakeLists.txt
+++ b/texturing/CMakeLists.txt
@@ -3,5 +3,5 @@ cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
 add_executable(vc_texture2 main.cpp UPointMapping.cpp plyHelper.cpp)
 target_link_libraries(vc_texture2 ${VC_LIBS} ${VC_REQ_LIBS} ${ITK_LIBRARIES})
 
-add_executable(vc_layering layering.cpp plyHelper.cpp meshUtils.cpp)
-target_link_libraries(vc_layering ${VC_LIBS} ${VC_REQ_LIBS} ${ITK_LIBRARIES})
\ No newline at end of file
+#add_executable(vc_layering layering.cpp plyHelper.cpp meshUtils.cpp)
+#target_link_libraries(vc_layering ${VC_LIBS} ${VC_REQ_LIBS} ${ITK_LIBRARIES})
